frontend/src/metabase/components/DateRelativeWidget/DateRelativeWidget.tsx

```diff
@@ -117,24 +120,18 @@ export class PredefinedRelativeDatePicker extends Component<PredefinedRelativeDa
         </section>
         {Object.keys(RELATIVE_SHORTCUTS).map(sectionName => (
           <section key={sectionName}>
-            <div
+            <fieldset
```

Comment:
Please see the comment in `frontend/src/metabase/components/DateRelativeWidget/DateRelativeWidget.module.css`.

This solves the issue with line cutoff in darktheme.